editor/editor_file_dialog.cpp

```diff
@@ -1218,6 +1222,7 @@ void EditorFileDialog::_update_drives(bool p_select) {
 
 void EditorFileDialog::_update_icons() {
 	// Update icons.
+	file->set_right_icon(item_list->get_theme_icon(SNAME("Search"), SNAME("EditorIcons")));
```

Comment:
Why this change?
This is not even search bar.

Reply:
Honestly I'm not sure why I did that. I remember thinking that it was a search bar, but clearly it isn't. I must have been tired or something lol.